js/src/Ice/ArrayUtil.d.ts

```diff
@@ -4,14 +4,30 @@
 
 declare module "ice" {
     namespace Ice {
+        /**
+         * Utility functions for arrays.
+         */
         class ArrayUtil {
+            /**
+             * Creates a shallow copy of the array.
+             * @param arr The array to clone.
+             * @returns A shallow copy of the array.
+             */
             static clone<T>(arr: T[]): T[];
```

Comment:
Can't we just use the array  `.slice()` method?

Reply:
Yes probably. I will check in a follow up PR.